vignettes/ExtendingGenomicRanges.Rmd

```diff
@@ -0,0 +1,102 @@
+---
+title: "**Extending Genomic Ranges**"
+author: "Michael Lawrence, Bioconductor Team"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+package: GenomicRanges
+vignette: >
+  %\VignetteIndexEntry{Extending Genomic Ranges}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+output:
+  BiocStyle::html_document
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+# Introduction
+
+The goal of *GenomicRanges* is to provide general containers for genomic
```

Comment:
*GenomicRanges* is italicized as well as a hyperlink in the original vignette so we can change this to [*GenomicRanges*](http://bioconductor.org/packages/release/bioc/html/GenomicRanges.html)

<img width="560" alt="Screen Shot 2022-11-05 at 12 23 25 AM" src="https://user-images.githubusercontent.com/105299892/200100540-a33c3b71-c406-4115-9f5a-032f52dbed49.png">